src/core/net/udp6.cpp

```diff
@@ -137,6 +137,36 @@ otError Udp::Socket::SendTo(Message &aMessage, const MessageInfo &aMessageInfo)
     return Get<Udp>().SendTo(*this, aMessage, aMessageInfo);
 }
 
+#if OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+otError Udp::Socket::JoinNetifMulticastGroup(otNetifIdentifier aNetifIdentifier, const Address &aAddress)
```

Comment:
Is it necessary to check if this is a multicast address?

Reply:
I think we'd better check it.